Update index.md - Обновление инструкции по работе скриншотов на CI

kasbyte opened the pull request:
Обновление инструкции по работе скриншотов на CI

https://t.me/testspro1c/111360

- Дополнен пункт 8: добавлен новый параграф о корректном завершении RDP-сессии через переключение на локальную консоль (чтобы избежать черного экрана при автотестах). 
- Предлагается рассмотреть возможность удаления второго параграфа пункта 8, так как информация частично дублируется.
- Пункт 13 можно переработать ссылаясь на пункт 8 для единой логики работы с RDP и скриншотами.
- Остальные части документа не изменялись.

kasbyte committed:
Update index.md

Обновление инструкции по работе скриншотов на CI

- Дополнен пункт 8: добавлен новый параграф о корректном завершении RDP-сессии через переключение на локальную консоль (чтобы избежать черного экрана при автотестах). 
- Четвертый подпункт в пункте 8 расширен с командами и пошаговой инструкцией.
- Предлагается рассмотреть возможность удаления второго параграфа пункта 8, так как информация частично дублируется.
- Пункт 13 можно переработать ссылаясь на пункт 8 для единой логики работы с RDP и скриншотами.
- Остальные части документа не изменялись.

Pr-Mex commented:
@kasbyte 
Привет.
Я бы тогда переформулировал, что есть выбор:
либо отказаться от использования RDP, либо использовать "переключая ее на локальную консоль"

kasbyte committed:
Обновление инструкции по работе скриншотов на CI

Дополнен пункт 8. Либо отказаться от использования RDP, либо использовать "переключая ее на локальную консоль"

Pr-Mex commented:
@kasbyte 
Ага, так хорошо

Pr-Mex commented:
@kasbyte 
Спасибо!

kasbyte commented:
После объединения изменений раздел на сайте открылся с ошибкой, нумерация нарушена. Материал по [ссылке](https://pr-mex.github.io/vanessa-automation/FAQ/#_1) не отображается корректно.

При этом содержимое в репозитории отображается правильно:
https://github.com/Pr-Mex/vanessa-automation/blob/develop/docs/FAQ/index.md

Похоже, что при публикации документации возникла проблема на стороне сборки сайта. Можете, пожалуйста, уточнить, как так получилось, и нужно ли что то поправить с моей стороны?

Pr-Mex commented:
@kasbyte 
Попробую сам поправить. Скорее всего дело в отступах.
